devtools/docker-compose.yml

```diff
@@ -1,22 +1,25 @@
-version: '2'
+version: '2.1'
```

Comment:
not sure this is useful

Reply:
https://docs.docker.com/compose/compose-file/compose-versioning/#version-21